src/Catalyst.Core.Lib/FileSystem/FileSystem.cs

```diff
@@ -62,10 +146,7 @@ public bool SetCurrentPath(string path)
                 var fullPath = Path.GetFullPath(path);
 
                 var dirInfo = new DirectoryInfo(fullPath);
-                if (!dirInfo.Exists)
-                {
-                    dirInfo.Create();
-                }
+                if (!dirInfo.Exists) dirInfo.Create();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)